packages/common/src/hooks/useSearchParameters/useSearchParameters.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { MemoryRouter, useSearchParams } from 'react-router-dom';
+import { Route, Routes } from 'react-router';
+
+import { useSearchParameters } from './useSearchParameters';
+
+const ShowParams = () => {
```

Comment:
yeah.. had a bit of a problem getting the history out in order to test and so bodged this in. Not great, but good enough? maybe.. 🤷

Reply:
Ah yes I can see how annoying that would be. Seems fine. I hope to write some tests of the actual pages and maybe we can put some tests there with more 'real' use cases.. though would still need to figure out how to get the history out, so maybe not!